server/lib/games.ts

```diff
@@ -158,3 +158,7 @@ export function getDiscardPile(lobbyNr: number): Card {
   const discardpile = getGameByLobby(lobbyNr).discardPileCards;
   return discardpile[discardpile.length - 1];
 }
+
+export function getNumberOfGames() {
+  return games[[].length];
```

Comment:
`[].length` returns `0` in any case. Thus, you access `games[0]`  (first element) every time.

Reply:
This doesn't work either, as it returns the whole game :(
` return games[[0].length]`
Returns a game object.

`retunr games[0].length`
Doesn't work, since it isn't an array and doesn't allow the length property.